package.json

```diff
@@ -66,6 +69,7 @@
         "react": "16.13",
         "react-dev-utils": "^10.2.1",
         "react-dom": "16.13",
+        "react-scripts": "^4.0.3",
```

Comment:
Can we add this to `storybook/package.json` instead ?

Reply:
Ok, saw the other comment now.

Reply:
Yeah I've tried. Still doesn't resolve successfully then.